homepairs/homepairsUI/src/Screens/Components/PrimaryContactInfo/PrimaryContactInfo.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react'; //* *For every file that uses jsx, YOU MUST IMPORT REACT  */
+import { StyleSheet, Text, View } from 'react-native';
+import strings from 'homepairs-strings';
+import { DarkModeInjectedProps } from '../WithDarkMode/WithDarkMode';
+import { HomePairsDimensions, Contact, } from 'src/state/types';
+import * as BaseStyles from 'homepairs-base-styles';
+import { HomePairFonts } from 'homepairs-fonts';
+
+export type PrimaryContactInfoProps = DarkModeInjectedProps & {
+    propertyManager: Contact;
+};
+
+const primaryContactStrings = strings.detailedPropertyPage.primaryContact;
+
+export default function PrimaryContactInfo(props: PrimaryContactInfoProps) {
+    const {
+        primaryColorTheme,
+        propertyManager,
+    } = props;
+
+    const {firstName, lastName, email} = propertyManager;
+    const styles = setStyles(primaryColorTheme);
+
+    function labeledItem(label: string, value: string) {
+        return (
+            <View style={styles.detailsWrapperStyle}>
+                <Text style={styles.detailsTitle}>{label}</Text>
+                <Text style={styles.cardDescription}>{value}</Text>
+            </View>
+        );
+    }
+
+    function ContactInfo() {
+        return (
+            <View style={styles.detailsWrapperStyle}>
+                {labeledItem(primaryContactStrings.name, (firstName + " " + lastName))}
+                {labeledItem(primaryContactStrings.email, (email + ""))}
+            </View>
+        );
+    }
+
+    /* Need the wrapper View so iOS can render the content properly. I don't know why it does this. */
+    return (
+        <View>
+            <View style={styles.container}>
+                <View style={styles.primaryContactContainer}>
+                    <Text style={styles.title}>{primaryContactStrings.title}</Text>
+                    <Text style={styles.subtitle}>{primaryContactStrings.subtitle}</Text>
+                </View>
+                {ContactInfo()}
+            </View>
+        </View>
+    );
+
+}
+
+function setStyles(colorTheme?: BaseStyles.ColorTheme) {
```

Comment:
FYI, we don't need these setStyles functions anymore since the DarkMode is being removed. You can simply just set the style sheet without need for a function.

Reply:
Caused problems getting rid of it / didn't cause problems not getting rid of it, so left it for this sprint since we're time sensitive